test/parallel/test-vm-property-not-on-sandbox.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const vm = require('vm');
+
+// These, admittedly contrived, examples test two edge cases of the vm module.
+//
+// Both GetterCallback and QueryCallback
+// explicitly check the global_proxy() if a property is
+// not found on the sandbox. In the following tests, the explicit check
+// inside the callback yields different results than deferring the
+// check until after the callback. The check is deferred, if the
```

Comment:
Treacherous comma?  It reads as if the "if the callbacks do not intercept" part is its own sentence.